src/hist.jl

```diff
@@ -227,7 +227,7 @@ function bkfe(gcounts, drv, bandwidth, range_x)
 
     ## Now combine weights and counts to obtain estimate
     ## we need P >= 2L+1L, M: L <= M.
-    P = 2^(ceil(log(M+L+1)/log(2)))
+    P = 2^(ceil(Int, log(M+L+1)/log(2)))
```

Comment:
Looks like this is just change of base to 2:
```suggestion
    P = 2^ceil(Int, log2(M+L+1))
```

Reply:
It seems this is just
```suggestion
    P = nextpow(2, M+L+1)
```